practice_app/api/views.py

```diff
@@ -1,3 +1,71 @@
-from django.shortcuts import render
+import requests
+import re
+from django.http import JsonResponse
+from . import api_keys
 
-# Create your views here.
+
+def google_scholar(request):
+    url = request.get_full_path().split('?')[-1] + '&'
```

Comment:
You can use request.GET to get dictionary

Reply:
It is fixed